packages/storybook/stories/Stateful Client/Adapters.stories.mdx

```diff
@@ -0,0 +1,136 @@
+
+## Setting Up Adapters
+
+To setup the adapters, you will need to provide the adapter with key metadata required.
+This includes most importantly an access token for your Azure Communication Services resource and context information for the call or chat thread you want to join.
+To create the adapters we will use the `createAzureCommunicationCallAdapter` and `createAzureCommunicationChatAdapter` helper methods.
+This methods are asynchronous so we will will `useEffect` to create them.
+
+### Calling Adapters
+
+To initialize the calling adapter, you will be required to provide a call locator with the information of the group or meeting you are trying to join.
+Group Id for Azure Communication Services take the shape of GUID generated by the developer to identify a call inside their tenant.
+Teams Meeting information can be acquired through [Microsoft Graph](https://docs.microsoft.com/en-us/graph/overview).
+
+```typescript
+import { CallAdapter, createAzureCommunicationCallAdapter } from '@azure/communication-react';
+
+const [callAdapter, setCallAdapter] = useState<CallAdapter>();
+const accessToken = 'INSERT ACCESS TOKEN';
+const displayName = 'INSERT DISPLAY NAME FOR LOCAL USER';
+const callLocator = { groupId: 'INSERT GROUP ID' };
+//CallLocator for Teams Meeting = const callLocator = {meetingLink: 'INSERT MEETING LINK'};
+
+useEffect(() => {
+  (async () => {
+    if (accessToken && callLocator && displayName) {
+      const createAdapter = async (): Promise<void> => {
+        setCallAdapter(await createAzureCommunicationCallAdapter(accessToken, callLocator, displayName));
+      };
+      createAdapter();
+    }
+  })();
+});
+```
+
+### Chat Adapter
+
+To initialize the chat adapter, you will be required to provide an endpoint url for the Azure Communication Services resource on [Azure Portal](https://portal.azure.com).
+In addition, you will need to provide a thread Id. Thread Id is provisioned from the Azure Communication Services by creating a new thread or adding the user to an existing thread.
+For more information on thread Id, see our [documentation](https://docs.microsoft.com/azure/communication-services/concepts/chat/concepts).
+
+```typescript
+import { ChatAdapter, createAzureCommunicationChatAdapter } from '@azure/communication-react';
+
+const [chatAdapter, setChatAdapter] = useState<ChatAdapter>();
+const accessToken = 'INSERT ACCESS TOKEN';
+const displayName = 'INSERT DISPLAY NAME FOR LOCAL USER';
+const endpointUrl = 'INSERT AZURE RESOURCE ENDPOINT';
+const threadId = 'INSERT THREAD ID FOR CHAT';
+
+useEffect(() => {
+  if (props) {
+    const createAdapter = async (): Promise<void> => {
+      setChatAdapter(await createAzureCommunicationChatAdapter(accessToken, endpointUrl, threadId, displayName));
+    };
+    createAdapter();
+  }
+});
```

Comment:
same comments here as the other example regarding useEffect dependencies, dispose fn call and being a snippet file